circuit_tutorials/plonky2/merkle_tree/circuit/src/lib.rs

```diff
@@ -0,0 +1,196 @@
+use plonky2::plonk::proof::ProofWithPublicInputs;
```

Comment:
quite a few full-line comments in this file extend past 100 chars.
Is there a tool you're using to detect over-long lines, etc? I tried `rustfmt` with limited success... it wrapped actual code just fine but did not have any effect on long single-line comments.
In any case let's use 100 as a maximum line length and wrap comments (manually if needed) as appropriate.